### PR TITLE
Update/functions

### PR DESCRIPTION
**Pull request by AlisonVilelaTenchi**

### CODE REVIEW

File: zanshinsdk/client.py [hunk 1]
```diff
@@ -38,6 +38,12 @@ class ScanTargetKind(str, Enum):
     GCP = "GCP"
     AZURE = "AZURE"
 
+class Rules(str, Enum):
```
Comment (asieira) [inline zanshinsdk/client.py]: You probably meant to write `Roles` here, right?

File: zanshinsdk/client.py [hunk 2]
```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
...
+    def delete_api_key(self, api_key_id: Union[UUID, str]) -> bool:
+        """
+        Deletes a given API key by its id, it will only work if the informed ID belongs to the current logged user.
+        <https://api.zanshin.tenchisecurity.com/#operation/deleteApiKey>
+        :param api_key_id: the ID of the API key
+        :return: a boolean if success
+        """
+        return self._request("DELETE", f"/me/apikeys/{validate_uuid(api_key_id)}").json()
+
+    def get_freshdesk_id(self) -> Dict:
```
Comment (asieira) [inline zanshinsdk/client.py:273]: Is this an operation our customers or integrations will need to use? Isn't this just for the SSO access to the Freshdesk web portal?

File: zanshinsdk/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-__version__ = "0.4.0"
+__version__ = "0.5.0"
```
Comment (asieira) [inline zanshinsdk/__init__.py]: Since we are doing breaking changes I think this version needs to be 1.0.0.

File: zanshinsdk/client.py [hunk 4]
```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
...
         :param organization_id: the ID of the organization
-        :param scan_target_ids: optional list of scan target IDs to list alerts from, defaults to all
-        :param states: optional list of states to filter returned alerts, defaults to all
-        :param severities: optional list of severities to filter returned alerts, defaults to all
-        :param page: page number, starting from 1
-        :param page_size: page size in number of alerts
-        :return:
+        :param name: the Name of the oranization
+        :param picture: the picture of the organization
+        :param email: the e-mail contact of the organization
```
Comment (asieira) [inline zanshinsdk/client.py:305]: What exactly do you expect here in this parameter? A URL? The binary contents of an image? What are the supported formats?

File: zanshinsdk/client.py [hunk 5]
```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
...
-            body['severities'] = [validate_class(x, AlertSeverity).value for x in severities]
-        return self._request("POST", "/alerts", body=body).json()
+        return self._request("PUT", f"/organizations/{validate_uuid(organization_id)}", body=body).json()
 
-    def iter_organization_alerts(self, organization_id: Union[UUID, str],
-                                 scan_target_ids: Optional[Iterable[Union[UUID, str]]] = None,
-                                 states: Optional[Iterable[AlertState]] = None,
-                                 severities: Optional[Iterable[AlertSeverity]] = None, page_size: int = 100) -> \
-            Iterator[Dict]:
+    def iter_members(self, organization_id: Union[UUID, str]) -> Iterator[Dict]:
```
Comment (asieira) [inline zanshinsdk/client.py:315]: I think `iter_organization_members` would be clearer and more in line with the other methods such as `iter_organization_alerts`.

Comment (asieira) [reply zanshinsdk/client.py:315]: Same for all of the other member methods.

File: zanshinsdk/client.py [hunk 6]
```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
...
+        """
+        Resend organization member invitation.
+        <https://api.zanshin.tenchisecurity.com/#operation/resendOrganizationInviteByEmail>
+        :param organization_id: the ID of the organization
+        :param email: the e-mail of the invited member
+        :return: a boolean if success
+        """
+        return self._request("POST", f"/organizations/{validate_uuid(organization_id)}/invites/{email}/resend").json()
+
+    def iter_followers(self, organization_id: Union[UUID, str]) -> Iterator[Dict]:
```
Comment (asieira) [inline zanshinsdk/client.py:412]: `iter_organization_followers`

File: zanshinsdk/client.py [hunk 7]
```diff
@@ -318,65 +580,166 @@ def create_scan_target(self, organization_id: Union[UUID, str], kind: ScanTarget
         return self._request('POST', f'/organizations/{validate_uuid(organization_id)}/scantargets',
                              body=body).json()
 
-    def start_scan_target(self, organization_id: Union[UUID, str], scan_target_id: Union[UUID, str]) -> Dict:
+    def get_scan_target(self, organization_id: Union[UUID, str], scan_target_id: Union[UUID, str]) -> Dict:
```
Comment (asieira) [inline zanshinsdk/client.py:583]: `get_organization_scan_target`

File: zanshinsdk/client.py [hunk 8]
```diff
@@ -318,65 +580,166 @@ def create_scan_target(self, organization_id: Union[UUID, str], kind: ScanTarget
...
+        :param organization_id: the ID of organization the scan target belongs to
+        :param scan_target_id: the ID of the scan target
+        :return: a dict representing the scan target
         """
-        yield from self._request("GET", f"/organizations/{validate_uuid(organization_id)}/following").json()
+        return self._request("POST",
+                             f"/organizations/{validate_uuid(organization_id)}/scantargets/{validate_uuid(scan_target_id)}/check").json()
 
-    def stop_following(self, organization_id: Union[UUID, str], following_id: Union[UUID, str]) -> bool:
+    def iter_scan(self, organization_id: Union[UUID, str], scan_target_id: Union[UUID, str]) -> Iterator[Dict]:
```
Comment (asieira) [inline zanshinsdk/client.py:641]: `iter_organization_scan_target_scans`

**Review by asieira — CHANGES_REQUESTED** [review]

File: zanshinsdk/client.py (hunk 2 above)
Comment (AlisonVilelaTenchi) [reply zanshinsdk/client.py:273]: yes, i will remove this function

**Review by asieira — APPROVED** [review]

**Pull request merged**